### ISSUE TITLE
How to fine tune and optimize the model

### ISSUE THREAD

**Issue by ncoll** [open-mmlab/mmocr#565]
Now I use PANet_CTW+SEG Identify my own picture，Some strings contain uppercase letters and numbers，eg: "LGKDG35918JKG";the question is when the string contains "S" may recognize "5", "W" to "vv" ,"J1" to "11",and also only one nums like "6" recognize "66",could you tell me how to fix this though my own data (I make some data to fine tune "seg" model but the acc became more badly),or should i modify the config  parameter when training?

**Comment by gaotongxiao** [comment]
Finetuning a model generally requires a much smaller learning rate than training from scratch. And it's hard to finetune the model for special cases, especially when your data is not large enough and your model is just likely to overfit.
